docs/docs/overview/quickstart/installation.md

```diff
@@ -30,20 +31,21 @@ To install Botpress, unzip the file you download somewhere on your computer. Mak
 - The right to read/write to the Botpress directory and subdirectories.
 
 ## Alternative - Starting Botpress Binaries
+<!-- as above. Shouldn't this be "Starting Botpress" isn't that required for both the quick install and the binaries? -->
 
-To start Botpress, all you have to do is double-click on the `bp` file in the directory you extracted Botpress.
+To start Botpress, double-click on the `bp` file in the directory you extracted Botpress.
 
-Alternatively, you can also start it from the terminal using the command:
+Alternatively, start Botpress from the terminal using the command:
 
 ```bash
 ./bp
 ```
```

Comment:
````suggestion
```

To see all the commands available, run `./bp --help`.

![CLI Start](/assets/cli-help.png)
````